docs/how-tos/entanglement_forging/freeze-orbitals.ipynb

```diff
@@ -81,64 +78,11 @@
     "H2_x = radius_2 * np.cos(np.pi / 180 * thetas_in_deg)\n",
     "H2_y = radius_2 * np.sin(np.pi / 180 * thetas_in_deg)\n",
     "\n",
-    "molecule = Molecule(\n",
-    "    geometry=[\n",
-    "        [\"O\", [0.0, 0.0, 0.0]],\n",
-    "        [\"H\", [H1_x, 0.0, 0.0]],\n",
-    "        [\"H\", [H2_x, H2_y, 0.0]],\n",
-    "    ],\n",
-    "    charge=0,\n",
-    "    multiplicity=1,\n",
+    "driver = PySCFDriver(\n",
```

Comment:
I'm curious why you decided to use `PySCFDriver` directly rather than `MoleculeInfo`, as I did in #59.

Reply:
Let me do that for consistency. I'm just so used to going straight to pyscfd

Reply:
So you need pyscf driver either way, and it takes the molecule info directly, so it's leaner just to use the pyscfd interface directly.

One thing to note is pyscfd defines `spin` as the `2*total spin` of system, and `MoleculeInfo` defines spin `multiplicity` as `2 * total spin + 1`. Just have to make that minor conversion between the interfaces in the cases of our how-tos